lib/live_debugger_refactor/api/system/process.ex

```diff
@@ -0,0 +1,66 @@
+defmodule LiveDebugger.API.System.Process do
+  @moduledoc """
+  This module provides wrappers for system functions that queries processes in the current application.
+  """
+  @callback initial_call(pid :: pid()) :: mfa() | nil
+  @callback state(pid :: pid()) :: {:ok, term()} | {:error, term()}
+  @callback list() :: [pid()]
+
+  @doc """
+  Wrapper for `Process.info/2` with some additional logic that returns the initial call of the process.
+  """
+  @spec initial_call(pid :: pid()) :: mfa() | nil
+  def initial_call(pid), do: impl().initial_call(pid)
```

Comment:
Wondering if this function shouldn't return some kind of error e.g. `{:error, :not_alive} 🤔 IMO it describes a bit better an outcome of this function